cmd/frontend/graphqlbackend/status_messages.go

```diff
@@ -0,0 +1,48 @@
+package graphqlbackend
+
+import (
+	"context"
+
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/backend"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/types"
+	"github.com/sourcegraph/sourcegraph/pkg/repoupdater"
+)
+
+func (r *schemaResolver) StatusMessages(ctx context.Context) ([]*StatusMessageResolver, error) {
+	var notifications []*StatusMessageResolver
+
+	// 🚨 SECURITY: Only site admins can see status messages.
+	if err := backend.CheckCurrentUserIsSiteAdmin(ctx); err != nil {
+		return nil, err
+	}
+
+	result, err := repoupdater.DefaultClient.StatusMessages(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	for _, rn := range result.Messages {
+		notifications = append(notifications, NewStatusMessage(&types.StatusMessage{
```

Comment:
It's confusing to have two names for the same concept.
```suggestion
		msgs = append(msgs, NewStatusMessage(&types.StatusMessage{
```

Reply:
Woops! That was a left over from the prototype :)